kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -145,6 +149,19 @@ def after_catalog_created(self, catalog):
         )
 
 
+def _check_is_known_ci_env():
+    # Most CI tools will set the CI environment variable to true
+    if os.getenv("CI") == "true":
+        return True
+
+    # Not all CI tools follow this convention, we can check through those that don't
+    for env_var_key in KNOWN_CI_ENV_VAR_KEYS:
+        if env_var_key in os.environ:
+            return True
+
+    return False
```

Comment:
Maybe

```suggestion
    # Not all CI tools follow this convention, we can check through those that don't
    return not set(KNOWN_CI_ENV_VAR_KEYS).isdisjoint(os.environ)
```

? (Didn't test this code, offering it as a suggestion)